### DISCUSSION TITLE
how do you change your kernver version

### DISCUSSION THREAD

**Discussion by buddyted** [3kh0/ext-remover#1248] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
is there any way to change it I'm just asking because I wanna downgrade SAFELY to 120 or further down then just 125 I'm open to opening it up and preforming surgery on it if I absolutely have to but is there anyway to I'm also on v125 with a kernver 4 dedde board

**Comment by ghost** [comment, marked as answer]
use kernel version switcher

https://github.com/S-PScripts/chromebook-utilities/blob/main/Exploits/KVS%20(Kernel%20Version%20Switcher)

**Reply by buddyted** [reply]
So will this unenroll the Chromebook just to double check

**Reply by schoolexploitkid** [reply]
it can only be done on chromebooks without FWMP aka unenrolled
